remoulade/helpers/validation.py

```diff
@@ -0,0 +1,14 @@
+from functools import update_wrapper
+
+from flask import request
+
+
+def validate_schema(schema):
+    def decorator(func):
+        def wrapper(*args, **kwargs):
+            schema().load(request.get_json() or {})
+            return func(*args, **kwargs)
+
+        return update_wrapper(wrapper, func)
```

Comment:
why do you need this?

Reply:
If we don't do this, as we use this decorator with multiple routes, flask will throw an error because other endpoint functions will have the same name and overwrite the function.